modules/tinymce/src/plugins/table/test/ts/browser/TableSectionApiTest.ts

```diff
@@ -334,6 +344,14 @@ describe('browser.tinymce.plugins.table.TableSectionApiTest', () => {
       it('TINY-6150: Switch header cell to body cell', () =>
         switchType(hook.editor(), headerCellContent, bodyContent, 'mceTableCellType', 'td', [ 'newcell', 'tablemodified' ], 'tr#one th')
       );
+
+      it('TINY-6150: Switch multiple body cells to header cells', () =>
```

Comment:
Note: I did intentionally use the original Jira for this as that's what it's more applicable to and is something we probably should have included back then.